src/containers/RegistrationContainer.js

```diff
@@ -28,43 +36,116 @@ const RegistrationContainer = props => {
     mattermostId,
     interestOptions,
     registrationError,
+    getMe,
+    userBirthdate,
   } = props
   const [nickname, setNickname] = useState('')
-  const [showAge, setShowAge] = useState(false)
+  const [showAge, setShowAge] = useState('')
   const [location, setLocation] = useState('')
+  const [showLocation, setShowLocation] = useState('')
   const [description, setDescription] = useState('')
   const [img, setImg] = useState(null)
   const [interests, setInterests] = useState([])
+  const [nextButtonActive, setNextButtonActive] = useState(true)
+
+  useEffect(() => {
+    getMe()
+    props.addUserToStateAction()
+  }, [])
+
+  // Change nextButtonActive value only if new value is different
+  const setNextButtonStatus = value => {
```

Comment:
Not completely sure why value is changed only when it's different.
also `if(value === true && !nextButtonActive)` is essentially just `if(value && !nextButtonActive)`

Reply:
If the `!nextButtonActive `and `nextButtonActive` parts of the condition are removed, the page keeps re-rendering ("Too many re-renders. React limits the number of renders to prevent an infinite loop."). Good point about the unnecessary comparison, I fixed that :)